core/internal/processlib/processlib_linux.go

```diff
@@ -0,0 +1,24 @@
+package processlib
+
+import (
+	"os"
+	"syscall"
+)
+
+const (
+	PRCTL_SYSCALL    = 157
+	PR_SET_PDEATHSIG = 1
```

Comment:
add a comment explaining the magic please

Reply:
added some notes, they are from linux header files.